app/views/learning_hours/_learning_hours_table.html.erb

```diff
@@ -0,0 +1,43 @@
+<div class="card card-container">
+  <div class="card-body">
+    <div class="table-wrapper table-responsive">
+      <table class="table learning-hours-list" id="all-learning-hours">
+        <thead>
+          <tr>
+            <th>Title</th>
+            <th>Learning Type</th>
+            <% if current_user.casa_org.learning_topic_active %>
+              <th>Learning Topic</th>
+            <% end %>
+            <th>Date</th>
+            <th>Time Spent</th>
+          </tr>
+        </thead>
+        <tbody>
+          <% learning_hours.order(occurred_at: :desc).each do |learning_hour| %>
```

Comment:
can we reuse a partial here instead?

Reply:
I see that `app/views/learning_hours/show.html.erb` has similar code so I could create a partial to avoid repeating some of the table data.